README.md

```diff
@@ -49,6 +49,18 @@ It's modelled after `UIImagePickerController`, which makes it a breeze to use.
 
 ## Installation
 
+### Swift Package Manager
+
+The [Swift Package Manager](https://swift.org/package-manager/) is a tool for automating the distribution of Swift code and is integrated into the `swift` compiler. It is in early development, but Alamofire does support its use on supported platforms.
```

Comment:
Were referencing `Alamofire` instead of `WeScan`.

Reply:
Oops